react-components/src/components/Architecture/Utilities.tsx

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { type BaseCommand } from '../../architecture/base/commands/BaseCommand';
+import { type RevealRenderTarget } from '../../architecture/base/renderTarget/RevealRenderTarget';
+import { RenderTargetCommand } from '../../architecture/base/commands/RenderTargetCommand';
+import { type IconType } from '@cognite/cogs.js';
+
+export function getIcon(command: BaseCommand): IconType | undefined {
+  if (command.icon === undefined) {
+    return undefined;
+  }
+  return command.icon as IconType;
+}
+
+export function getTooltipPlacement(isHorizontal: boolean): 'top' | 'right' {
+  return isHorizontal ? 'top' : 'right';
```

Comment:
Maybe is better to define a type for this.
Perhaps:

```typescript
type TooltipPlacementType = 'top' | 'right';
``` 

And also create/use a type file to store all these new type definitions there

Reply:
Fixed